services/src/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v1/SwitchController.java

```diff
@@ -263,6 +265,20 @@
         return switchService.syncRules(switchId);
     }
 
+    /**
+     * Synchronize (install) missing flows that should be on the switch but exist only in neo4j.
+     * Optionally removes excess rules from the switch.
+     *
+     * @return the synchronization result.
+     */
+    @ApiOperation(value = "Synchronize rules on the switch", response = RulesSyncResult.class)
+    @PostMapping(path = "/{switch_id}/synchronize")
```

Comment:
If we follow [API guidelines](https://github.com/telstra/open-kilda/blob/develop/docs/guidelines/api-guidelines.md) written by Sergii, then there should be a PATCH method. Please correct me if I'm wrong